src/Bonsai.ML.Torch/LinearAlgebra/Eig.cs

```diff
@@ -0,0 +1,29 @@
+using System;
+using System.ComponentModel;
+using System.Reactive.Linq;
+using static TorchSharp.torch;
+
+namespace Bonsai.ML.Torch.LinearAlgebra
+{
+    /// <summary>
+    /// Computes the eigenvalue decomposition of a square matrix if it exists.
+    /// </summary>
+    [Combinator]
+    [Description("Computes the eigenvalue decomposition of a square matrix if it exists.")]
+    [WorkflowElementCategory(ElementCategory.Transform)]
+    public class Eig
```

Comment:
```suggestion
    public class Eig
```

Wow, these names are really terse, I just feel it would be really hard to have them standout for a non-technical audience, but I understand the difficult line we are threading here...

Not sure which way to go tbh, an alternative would be to go with Math.NET terminology, e.g. https://iridium.mathdotnet.com/api/MathNet.Numerics.LinearAlgebra/EigenvalueDecomposition.htm